server/pageRenderer.js

```diff
@@ -58,7 +58,7 @@ export default function pageRenderer({
   const isSSR = body === '' ? 'false' : 'true';
   return `
     <!DOCTYPE html>
-    <html>
+    <html data-theme="light">
```

Comment:
```suggestion
    <html data-theme=${serialize(selectCurrentUser(state).selectedTheme || "light")}>
```

Reply:
This will SSR with the correct value from the user :tm:

Reply:
With 

```js
import { selectCurrentUser } from 'app/reducers/auth';
```

Reply:
@odinuge Hm, when I tested your change in ssr:staging, it breaks ssr upon first load, because selectCurrentUser is passed undefined. I tried to fix this with an if test, which worked, but got flow errors:

![image](https://user-images.githubusercontent.com/26925695/78554317-3822ce80-780b-11ea-8231-75c567870111.png)

It seemed to work really well tho, when the page is allready loaded, and you refresh the page. It removes the white flash that occurs before the methods in ThemeUtils are called. Any suggestings on how this can be solved? :thinking:

Reply:
Ahh.

You should be able to use:

```js
import {isEmpty} from 'lodash';

const getIt = () =>{
  if (!isEmpty(state)){
    return selectCurrentUser(state).selectedTheme;
  }
}
```

And use it like

```
serialize(getIt() || 'light');
```

Reply:
Thank you for your continued follow up! Tested this on ssr, and it worked really well.

Reply:
I thought about having an auto option, which would be great for dark mode power users. It is a nice-to-have feature, and I will probably implement this soon. This was not included in v1, because I wanted to keep it as simple as possible for the first release. :dark_sunglasses: